CMakeLists.txt

```
@@ -21,6 +21,7 @@ find_package(LLBuild QUIET)
 find_package(ArgumentParser CONFIG REQUIRED)
 find_package(SwiftCollections QUIET)
 find_package(SwiftSystem CONFIG REQUIRED)
+include(${SwiftSyntax_DIR}/cmake/SwiftSyntaxTargets.cmake)
```

This needs an associated `find_package`.  Currently this is relying on someone happening to specify a variable that is similarly named and uses that to load a file that may or may not be named the same.

```suggestion
find_package(SwiftSyntax CONFIG REQUIRED)
```